packages/graphql-codegen/src/generateEnumsGraphql.ts

```diff
@@ -18,3 +18,23 @@ export async function generateEnumsGraphql(enums: EnumRows): Promise<CodeGenFile
 
   return { name: "../../schema/enums.graphql", overwrite: true, contents: formatted };
 }
+
+/** Maps db types, i.e. `int`, to GraphQL types, i.e. `Int`. */
+function mapSimpleDbType(dbType: string): string {
```

Comment:
Seems like we should have this already, is it:

```
function mapFieldTypeToGraphQLType(type: string | SymbolSpec): string | SymbolSpec {
```

Oh, I see, that's from TS type --> db type.

We could export this from `codegen` instead? I believe `graphql-codegen` already accesses exported functions from that package. Like the `config`.

Or basically someone tie together the existing code function that goes db type --> TS type and then the existing TS type --> GQL type.

Reply:
Done, TY 👍